app/models/api_authorization/constraint.rb

```diff
@@ -0,0 +1,17 @@
+module ApiAuthorization
+  ##
+  # Constraint used gate API access to annotations
+  class Constraint
+    def matches?(request)
+      return true if request.get? || valid_api_key?(request)
+      raise ApiAuthorization::Unauthorized
+    end
+
+    def valid_api_key?(request)
+      ActiveSupport::SecurityUtils.secure_compare(
+        ::Digest::SHA256.hexdigest(request.headers['Authorization'].to_s),
```

Comment:
I'm trying to figure out if a bare token in the `Authorization` header is 👌 , or if a type specification (e.g. `Bearer`) is required.

Reply:
Some discussion here: https://community.apigee.com/questions/7498/how-to-comply-with-http-standard-when-putting-the.html?childToView=7514#answer-7514

/shrug ?